csharp/tutorials/nullable-reference-migration/finished/SimpleFeedReader/Services/NewsService.cs

```diff
@@ -73,8 +73,6 @@ public NewsStoryProfile()
             // Create the AutoMapper mapping profile between the 2 objects.
             // ISyndicationItem.Id maps to NewsStoryViewModel.Uri.
             CreateMap<ISyndicationItem, NewsStoryViewModel>()
-                .ForCtorParam("published", opt => opt.MapFrom(src => src.Published))
```

Comment:
@lbargaoanu Does AutoMapper manage the mapping when case is different?

Reply:
Yes, AM is case insensitive.

Reply:
And in fact, if there were a mismatch, `AssertConfigurationIsValid` would report it.